packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -95,7 +95,12 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
                 <ParticipantList
                   {...participantListProps}
                   onRenderAvatar={(userId, options) => (
-                    <AvatarPersona userId={userId} {...options} dataProvider={onFetchAvatarPersonaData} />
+                    <AvatarPersona
+                      data-ui-id="chat-composite-participant-custom-avatar"
```

Comment:
BTW, should this come from some indetifierProvider or something?

Reply:
So this is a bit tricky here.
`IdentityProvider` is a part of the react-components package and contains ids for pure components.
If we add ids used inside composite to the provider as well, it muddies its scope

Reply:
What can be done here is the component based identityprovider can be extended and recreated for composites.

Reply:
So there are multiple places in composite where data-ui-ids have been hardcoded.
We can go ahead like this for now and have a separate task for cleaning data-ui-ids in composites ;)

Reply:
Sounds good, we might want to find a consistent way all across composite and component in the end, but sounds it is more complicated than this PR itself